api_types/src/api/networks.rs

```diff
@@ -3,8 +3,8 @@ use serde::{Deserialize, Serialize};
 #[derive(Debug, Deserialize, Serialize, Clone, PartialEq)]
 #[cfg_attr(feature = "utoipa", derive(utoipa::ToSchema))]
 pub enum ApiNetworkType {
-    Evm { node_url: String, chain_id: u64 },
-    Stardust { node_url: String },
+    Evm { node_url: Vec<String>, chain_id: u64 },
```

Comment:
Should we also rename the variable to indicate that there are multiple URLs?
```suggestion
    Evm { node_urls: Vec<String>, chain_id: u64 },
```

Reply:
Done 👍